moto/swf/exceptions.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import unicode_literals
+
+from boto.exception import JSONResponseError
+
+
+class SWFClientError(JSONResponseError):
```

Comment:
I must admit I cannot wrap my head around this one: when I use `werkzeug.exceptions.HTTPException` as a base class for my exceptions, the tests now say that the calls raise a `boto.exception.SWFResponseError`, thought I didn't use it anywhere explicitly. Not sure if it's OK or not, I'd prefer discussing it in another PR if you're OK, so I can focus on it better and not pollute the commit history with trials/errors here.